pyproject.toml

```diff
@@ -1,3 +1,8 @@
 [build-system]
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+addopts = [
+    "--import-mode=importlib",
+]
```

Comment:
Pytest recommends setting this config. They wish it was the default but it's too breaking to change.